<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogATV12</class>
 <widget class="QDialog" name="DialogATV12">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ATV12</string>
  </property>
  <widget class="QWidget" name="layoutWidget113">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>51</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="LFT1">
      <property name="text">
       <string>Привод 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LFT3">
      <property name="text">
       <string>Привод 3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="LFT4">
      <property name="text">
       <string>Привод 4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="LFT5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="LFT6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="LFT2">
      <property name="text">
       <string>Привод 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget9339">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>65</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Привод 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Привод 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Привод 3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Привод 4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LFT</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>51</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="freq1">
      <property name="text">
       <string>Привод 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="freq3">
      <property name="text">
       <string>Привод 3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="freq4">
      <property name="text">
       <string>Привод 4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="freq5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="freq6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="freq2">
      <property name="text">
       <string>Привод 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>частота</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>51</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="curr1">
      <property name="text">
       <string>Привод 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="curr3">
      <property name="text">
       <string>Привод 3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="curr4">
      <property name="text">
       <string>Привод 4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="curr5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="curr6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="curr2">
      <property name="text">
       <string>Привод 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ток</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>51</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="stat1">
      <property name="text">
       <string>Привод 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="stat3">
      <property name="text">
       <string>Привод 3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="stat4">
      <property name="text">
       <string>Привод 4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="stat5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="stat6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Привод 6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="stat2">
      <property name="text">
       <string>Привод 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>статус</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget929292">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>151</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="start1">
      <property name="text">
       <string>Пуск</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="stop1">
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="start2">
      <property name="text">
       <string>Пуск</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="stop2">
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="start3">
      <property name="text">
       <string>Пуск</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="stop3">
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="start4">
      <property name="text">
       <string>Пуск</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="stop4">
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
